### PR TITLE
Add collection name to error message

### LINKED ISSUE #14: Ambiguous error message
Hi zalando,

I'm using the Arch Linux distribution and don't use a desktop environment, just a window manager (AwesomeWM). I ran into an issue when using a tool that included go-keyring where it was returning "ERROR: failed to unlock correct collection". I didn't know what specific collection it was referring to, so I had to trace it back to here and saw that you included the helpful information in your readme. I believe it would be better to include the name of the collection you are trying to unlock so if anyone runs into the same problem, they don't have to track it down.

Thanks,
Joey

### PR DESCRIPTION
**Pull request by mikkeloscar**
Adds collection name to the error message so it's easier to debug.

Fix #14

### CODE REVIEW

File: secret_service/secret_service.go [hunk 1]
```diff
@@ -93,7 +93,7 @@ func (s *SecretService) Unlock(collection dbus.ObjectPath) error {
 	}
 
 	if len(unlocked) != 1 || unlocked[0] != collection {
-		return fmt.Errorf("failed to unlock correct collection")
+		return fmt.Errorf("failed to unlock correct collection '%s'", collection)
```
Comment (szuecs) [inline secret_service/secret_service.go]: AFAIK you should use %s only for strings, and default to %v

Comment (mikkeloscar) [inline secret_service/secret_service.go]: collection is a string in the end :)
I changed it to `%v`.

**Comment by szuecs** [comment]
:+1:

**Comment by mikkeloscar** [comment]
:+1:

**Pull request merged**